backends/LxcBackend.py

```diff
@@ -339,8 +345,12 @@ def update(self):
                 print("Container seems to have failed to start (no IP)", file=sys.stderr)
                 sys.exit(1)
 
-            ret = c.attach_wait(lxc.attach_run_command, ["env"] + ["MILXCGUARD=TRUE"] + [
-                            getInterpreter(filesdir), "/mnt/lxc/" + path + "/update.sh"], env_policy=lxc.LXC_ATTACH_CLEAR_ENV)
```

Comment:
same as previously, I think we can fit better paramters and subparams

Reply:
Indeed, here too it is possible to put everything in one line.
But I thought that in terms of visibility, it would be better to air the whole thing out a bit.
So, what do you find most appropriate?

Reply:
I think "[getInterpreter(filesdir),
                "/mnt/lxc/" + path + "/update.sh"]" should not be split on two lines, it is the same "atom", the rest is ok

Reply:
Fixed.